docs/src/content/docs/fr/reference/overrides.md

```diff
@@ -0,0 +1,374 @@
+
+Un tableau de toutes les en-têtes Markdown extraites de la page courante.
+Utilisez [`toc`](#toc) à la place si vous souhaitez construire une table des matières qui respecte les options de configuration de Starlight.
+
+#### `lastUpdated`
+
+**Type:** `Date | undefined`
+
+Un objet JavaScript de type `Date` représentant la date de dernière mise à jour de cette page si cette fonctionnalité est activée.
+
+#### `editUrl`
+
+**Type:** `URL | undefined`
+
+Un objet `URL` de l'adresse où cette page peut être modifiée si cette fonctionnalité est activée.
+
+---
+
+## Composants
+
+### Métadonnées
+
+Ces composants sont utilisés dans l'élément `<head>` de chaque page.
+Ils ne doivent inclure que des [éléments autorisés à l'intérieur de `<head>`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/head).
+
+#### `Head`
+
+**Composant par défaut:** [`Head.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/Head.astro)
+
+Composant utilisé à l'intérieur de l'élément `<head>` de chaque page.
+Inclut des balises importantes comme `<title>` et `<meta charset="utf-8">`.
+
+Redéfinissez ce composant en dernier recours.
+Préférez l'option [`head`](/fr/reference/configuration#head) de la configuration de Starlight si possible.
+
+#### `ThemeProvider`
+
+**Composant par défaut:** [`ThemeProvider.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/ThemeProvider.astro)
+
+Composant utilisé à l'intérieur de l'élément `<head>` qui configure la prise en charge du thème sombre/clair.
+L'implémentation par défaut inclut un script en ligne et un élément `<template>` utilisé par le script situé dans [`<ThemeSelect />`](#themeselect).
+
+---
+
+### Accessibilité
+
+#### `SkipLink`
+
+**Composant par défaut:** [`SkipLink.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/SkipLink.astro)
+
+Composant utilisé comme premier élément à l'intérieur du `<body>` qui relie au contenu principal de la page pour des raisons d'accessibilité.
+L'implémentation par défaut est masquée jusqu'à ce qu'il reçoive le focus d'un utilisateur utilisant la navigation au clavier.
+
+---
+
+### Mise en page
+
+Ces composants sont responsables de la mise en page des composants de Starlight et de la gestion des vues pour différents points d'arrêt.
+Redéfinir ceux-ci implique une complexité significative.
+Lorsque cela est possible, préférez redéfinir un composant de plus bas niveau.
+
+#### `PageFrame`
+
+**Composant par défaut:** [`PageFrame.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/components/PageFrame.astro)
+
+Composant de mise en page contenant la plupart du contenu de la page.
+L'implémentation par défaut configure la mise en page de l'en-tête, de la barre latérale et du contenu principal et inclut des emplacements (slots) nommés `header` et `sidebar` en plus de l'emplacement par défaut pour le contenu principal.
```

Comment:
Using the same syntax as the [Astro Docs](https://docs.astro.build/fr/core-concepts/astro-components/#les-emplacements-slots-nomm%C3%A9s).